src/components/ConversationIcon.vue

```diff
@@ -155,6 +156,21 @@ export default {
 			return this.$store?.getters.getMainContainerSelector()
 		},
 
+		iconClass() {
+			if (this.item.token) {
+				// Existing conversations use the /avatar endpoint… Always!
```

Comment:
![Screenshot from 2023-04-04 14-44-06](https://user-images.githubusercontent.com/93392545/229796654-7c3e2d8a-3884-49a1-be2e-e3b1f966efdc.png)

Somehow not

Reply:
Hmm interesting, seems like the token is not populated for open conversations, will have a look

Reply:
Works as a patch for the moment:
![image](https://user-images.githubusercontent.com/93392545/229803886-65461391-3ac3-4971-bf08-263ef8c5ec29.png)

constants are here: https://github.com/nextcloud/spreed/commit/eae9143fd24151a93e33213950f2e43eb025cc6b

Reply:
We don't want those constants. The frontend should not generate avatars directly anymore when it's conversations.

Reply:
In that case everything should work if we just remove `<template>` tags, because group icon is show up as well for me
Other additions are unneccessary